config/initializers/time_formats.rb

```diff
@@ -0,0 +1,2 @@
+Time::DATE_FORMATS[:long_no_time] = '%B %d, %Y'
```

Comment:
Can we have `:date_only_long` instead? Specify what the symbol has rather than what it lacks.